WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -173,12 +173,12 @@ public void onBindViewHolder(GridViewHolder holder, int position) {
             if (isLocalFile) {
                 loadLocalImage(media.getFilePath(), holder.imageView);
             } else {
-                WordPressMediaUtils.loadNetworkImage(getBestImageUrl(media), holder.imageView, mImageLoader);
```

Comment:
Same as above, I think `mImageLoader` should be re-added once that PR is done. Since the new 
Anyway dropped it for now!